test/Vesting.test.js

```diff
@@ -0,0 +1,288 @@
+        expectEvent(tx, "Withdraw", { to: acc2, amountTokens: ether('490') });
+      });
+      it("take tokens AFTER 600 MINUTES by acc2", async () => {
+        let balanceBefore = await instanceToken.balanceOf(acc2);
+        expect(balanceBefore.toString()).to.be.equal(ether('990').toString());
+        let investor1 = await instanceVesting.listOfBeneficiaries(acc2);
+        expect(investor1[1].toString()).to.be.equal(ether('990').toString());
+        await time.increase(time.duration.minutes(300));
+        let tx = await instanceVesting.withdrawTokens({ from: acc2 });
+        balanceTokens = await instanceToken.balanceOf(acc2);
+        expect(balanceTokens.toString()).to.be.equal(ether('1000').toString());
+        let investorr1 = await instanceVesting.listOfBeneficiaries(acc2);
+        expect(investorr1[1].toString()).to.be.equal(ether('1000').toString());
+        expectEvent(tx, "Withdraw", { to: acc2, amountTokens: ether('10') });
+      });
+      it("take tokens AFTER 600 MINUTES by acc4", async () => {
+        let balanceBefore = await instanceToken.balanceOf(acc4);
+        expect(balanceBefore.toString()).to.be.equal(ether('450').toString());
+        let investor3 = await instanceVesting.listOfBeneficiaries(acc4);
+        expect(investor3[1].toString()).to.be.equal(ether('450').toString());
+        await time.increase(time.duration.minutes(300));
+        let tx = await instanceVesting.withdrawTokens({ from: acc4 });
+        balanceTokens = await instanceToken.balanceOf(acc4);
+        expect(balanceTokens.toString()).to.be.equal(ether('4000').toString());
+        let investorr3 = await instanceVesting.listOfBeneficiaries(acc4);
+        expect(investorr3[1].toString()).to.be.equal(ether('4000').toString());
+        expectEvent(tx, "Withdraw", { to: acc4, amountTokens: ether('3550') });
+      });
+    });
+
+    describe("withdrawTokens function - false", async () => {
+      it("withdrawTokens function - call after withdraw all tokens by acc2", async () => {
+        let balanceTokensBefore = await instanceToken.balanceOf(acc2);
+        expect(balanceTokensBefore.toString()).to.be.equal(ether('1000').toString())
+        await expectRevert(instanceVesting.withdrawTokens({ from: acc2 }), "Error : not enougth tokens");
+        let balanceTokens = await instanceToken.balanceOf(acc2);
+        expect(balanceTokens.toString()).to.be.equal(ether('1000').toString())
+      });
+      it("withdrawTokens - Error : not enougth tokens ", async () => {
+        let balanceBefore = await instanceToken.balanceOf(owner);
+        expect(balanceBefore.toString()).to.be.equal(ether('93000').toString())
+        await expectRevert(instanceVesting.withdrawTokens({ from: owner }), "Error : not enougth tokens");
+        let balanceTokens = await instanceToken.balanceOf(owner);
+        expect(balanceTokens.toString()).to.be.equal(ether('93000').toString())
+      });
+    });
+
```

Comment:
add tests when you claim all tokens and then your address will add to investors second time claim tokens again )